api/jest.config.js

```diff
@@ -0,0 +1,5 @@
+/** @type {import('ts-jest').JestConfigWithTsJest} */
+module.exports = {
+  preset: 'ts-jest',
+  testEnvironment: 'node',
+};
```

Comment:
Could you explain the addition of this configuration?

Reply:
(answered in person)